packages/java/src/browser/java-protocol.ts

```diff
@@ -66,3 +66,7 @@ export enum CompileWorkspaceStatus {
 export namespace CompileWorkspaceRequest {
     export const type = new RequestType<boolean, CompileWorkspaceStatus, void, void>('java/buildWorkspace');
 }
+
+export namespace ExecuteClientCommand {
+    export const type = new RequestType<ExecuteCommandParams, undefined, void, void>('workspace/executeClientCommand');
```

Comment:
Is the type really `undefined` or did you do this to avoid the `no-any` rule?

Because if the any is necessary, you can add a comment above it, such as:
```ts
// tslint:disable-next-line:no-any
```

Reply:
Made `undefined` to silence the compiler... felt it's ok because we don't care about the return type at the moment